src/app/core/layout/navigation/navigation.component.ts

```diff
@@ -41,6 +41,7 @@ export class NavigationComponent implements OnInit {
   protected readonly navigationItems = [
     { link: '/', label: 'Blog' },
     { link: '/principles', label: 'Principles' },
+    { link: '/authors', label: 'Authors' },
```

Comment:
Let's not add it to the main navigation, please
```suggestion
```

Reply:
I've already removed on my local branch